automation/SetUp.ps1

```diff
@@ -0,0 +1,17 @@
+$SwitchName = "External VM Switch" 
```

Comment:
can all these 3 be parameters passed to the script with a default value?

Reply:
I'd like to execute this using something like: `Setup.ps1 -SwitchName "External VM Switch" -ISOFilePath "~\Downloads\..." -VMName "minikube-windows-1"`